certbot/certbot/_internal/main.py

```diff
@@ -1779,7 +1779,7 @@ def reconfigure(config: configuration.NamespaceConfig,
     for param in ('account', 'server',):
         if getattr(lineage_config, param) != renewalparams.get(param):
             msg = ("Using reconfigure to change the ACME account or server is not supported. "
-                   "If you would like to do so, use certonly with the --force-renewal flag instead "
+                   "If you would like to do so, use renew with the --force-renewal flag instead "
                    "of reconfigure. Note that doing so will count against any rate limits. For "
                    "more information on this method, see "
                    "https://certbot.org/certonly-reconfiguration")
```

Comment:
I'm not sure if this is ready for another review, but just to make sure you're not waiting on me here, I'll chime in and say that I think this link should now be
```suggestion
                   "https://certbot.org/renew-reconfiguration")
```
based on the conversations I saw.

Other than that, this PR LGTM! I'm OK with whatever order of operations between getting this link live and merging this PR that you prefer.

Reply:
Weird, I thought I changed that! Thanks for catching that.

Reply:
Ha, I changed it locally but didn't save the file before committing 🤦